pkg/controller/installation/installation_controller.go

```diff
@@ -140,6 +140,7 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 				log.Info("Error updating Installation resource. Requeue and retry.")
 				return reconcile.Result{
 					Requeue: true,
+					RequeueAfter: time.Second * 10,
```

Comment:
seems arbitrary?

Reply:
yeah, we can change it, it just has to have a value, or else it does backoff logic, and eventually doesn't requeue it for ages.